sormas-backend/src/main/java/de/symeda/sormas/backend/action/Action.java

```diff
@@ -0,0 +1,159 @@
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *******************************************************************************/
+package de.symeda.sormas.backend.action;
+
+import static de.symeda.sormas.api.EntityDto.COLUMN_LENGTH_BIG;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.ManyToOne;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import de.symeda.auditlog.api.Audited;
+import de.symeda.sormas.api.action.ActionContext;
+import de.symeda.sormas.api.action.ActionPriority;
+import de.symeda.sormas.api.action.ActionStatus;
+import de.symeda.sormas.backend.common.AbstractDomainObject;
+import de.symeda.sormas.backend.event.Event;
+import de.symeda.sormas.backend.user.User;
+
+@Entity
+@Audited
+public class Action extends AbstractDomainObject {
+	private static final long serialVersionUID = -4754578341242164661L;
+
+	public static final String TABLE_NAME = "action";
+	public static final String DESCRIPTION = "description";
+	public static final String REPLY = "reply";
+	public static final String REPLYING_USER = "replyingUser";
+	public static final String CREATOR_USER = "creatorUser";
+	public static final String PRIORITY = "priority";
+	public static final String DATE = "date";
+	public static final String EVENT = "event";
+	public static final String STATUS_CHANGE_DATE = "statusChangeDate";
+	public static final String ACTION_CONTEXT = "actionContext";
+	public static final String ACTION_STATUS = "actionStatus";
+
+	private ActionContext actionContext;
+	private Event event;
+
+	private ActionPriority priority;
+	private Date date;
+	private ActionStatus actionStatus;
+	private Date statusChangeDate;
+
+	private User creatorUser;
+	private String description;
+	private String reply;
+	private User replyingUser;
+
+	@Enumerated(EnumType.STRING)
+	public ActionContext getActionContext() {
+		return actionContext;
+	}
+
+	public void setActionContext(ActionContext actionContext) {
+		this.actionContext = actionContext;
+	}
+
+	@ManyToOne(cascade = {})
```

Comment:
cascade = {} is the default and therefore unnecessary

Reply:
OK